### PR TITLE
Feature/project

### PR DESCRIPTION
**Pull request by davidebukali**
Maybe this needs to use partial for related content from events before its merged. Current partial is affected because it uses page content type as a tag.

### CODE REVIEW

File: di_website/templates/includes/cards/card-related.html [hunk 1]
```diff
@@ -6,7 +6,8 @@
         <span class="card__tag">{{tag}}</span>
     {% endif %}
     {% image item.hero_image max-400x250 as hero_img %}
-    <span class="card__tag">{{ tag }}</span>
+    {% comment %} TODO: Use reusable partial for related content to load correct tags {% endcomment %}
+    <span class="card__tag">{{ item.content_type }}</span>
```
Comment (edwinmp) [inline di_website/templates/includes/cards/card-related.html]: Be careful when editing shared content ... has a larger impact.
In this case, we want to see a simple tag here... 'Blog', 'News', 'Event' e.t.c ... `content_type` doesn't fit.

File: di_website/project/models.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+from modelcluster.fields import ParentalKey
+
+from wagtail.admin.edit_handlers import (
+    StreamFieldPanel,
+    InlinePanel,
+    PageChooserPanel,
+    MultiFieldPanel,
+    FieldPanel
+)
+
```
Comment (edwinmp) [inline di_website/project/models.py]: Remove blank line

File: di_website/project/models.py [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+from wagtail.images.edit_handlers import ImageChooserPanel
+
```
Comment (edwinmp) [inline di_website/project/models.py]: Remove blank line

File: di_website/project/models.py [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+    TextBlock,
+    URLBlock
+)
+
+from di_website.common.base import hero_panels
+from di_website.common.mixins import BaseStreamBodyMixin, HeroMixin, OtherPageMixin
+from di_website.common.blocks import BaseStreamBlock
+from di_website.common.constants import MAX_RELATED_LINKS
+
+class ProjectPage(BaseStreamBodyMixin, HeroMixin, Page):
```
Comment (edwinmp) [inline di_website/project/models.py]: Conventionally, 2 blank ines before class definition.

File: di_website/project/models.py [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+    ]
+
+    parent_page_types = [
+        'FocusAreasPage'
+    ]
+
+    class Meta():
+        verbose_name = 'Project Page'
+
+class ProjectPageRelatedLink(OtherPageMixin):
```
Comment (edwinmp) [inline di_website/project/models.py]: Another blank line

File: di_website/project/models.py [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+    page = ParentalKey(
+        Page,
+        related_name='project_related_links',
+        on_delete=models.CASCADE
+    )
+    panels = [
+        PageChooserPanel('other_page')
+    ]
+
+class FocusAreasPage(BaseStreamBodyMixin, HeroMixin, Page):
```
Comment (edwinmp) [inline di_website/project/models.py]: Another blank line before

File: di_website/project/models.py [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+        on_delete=models.CASCADE
+    )
+    first_project_page = models.ForeignKey(
+        'wagtailcore.Page',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+',
+        verbose_name='First Project Page',
+        help_text='A page to link to projects in Focus Areas Section'
```
Comment (edwinmp) [inline di_website/project/models.py]: Oh, that's what this is for? ... hmm, nope ... what if I want to like 3 or more projects?

Comment (edwinmp) [reply di_website/project/models.py]: Opt for a more extensible solution.

File: di_website/project/models.py [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+    class Meta():
+        verbose_name = 'Focus Areas Page'
+
+    parent_page_types = ['home.HomePage']
+
+    content_panels = Page.content_panels + [
+        hero_panels(),
+        StreamFieldPanel('body'),
+        MultiFieldPanel([
+            InlinePanel('focus_areas_page_link', label="Focus Areas Section", max_num=6)
```
Comment (edwinmp) [inline di_website/project/models.py]: Good idea 👍

File: di_website/project/models.py (hunk 2 above)
Comment (davidebukali) [reply di_website/project/models.py]: Removed it

File: di_website/project/models.py (hunk 3 above)
Comment (davidebukali) [reply di_website/project/models.py]: Removed it

File: di_website/project/models.py (hunk 4 above)
Comment (davidebukali) [reply di_website/project/models.py]: Added extra line

File: di_website/project/models.py (hunk 5 above)
Comment (davidebukali) [reply di_website/project/models.py]: Added extra line

File: di_website/project/models.py (hunk 6 above)
Comment (davidebukali) [reply di_website/project/models.py]: Added blank line

File: di_website/project/models.py (hunk 7 above)
Comment (davidebukali) [reply di_website/project/models.py]: Now using PageChooserBlock for multiple pages

**Pull request merged**